### ISSUE TITLE
Class presentations

### ISSUE THREAD

**Issue by jstac** [jstac/quantecon_nyu_2016#16]
I would like to get presentations by class participants started soon --- ideally from March 11.  The presentations are not long and don't have to be formal.  The main idea is to teach us something that you find interesting and you believe is potentially useful.  Some suggestions are here:

https://github.com/jstac/quantecon_nyu_2016#notes-on-class-presentations

Please send me an email to discuss topic and date.  I won't continue to remind you about this course requirement so it's up to you to schedule your talk.

**Comment by vgregory757** [comment]
Hi John,

Can I present sometime in early April (maybe April 1)? I was thinking of
talking about either the machine learning library, or I found this cool
plotting library that allows you to make dynamic interactive charts:
https://plot.ly/python/

Also, what does the content of the presentation need to be like? Just
explaining main features or also presenting some application that you've
put together?

Thanks,
Victoria

**Comment by jstac** [comment]
@vgregory757 Great, let's set your presentation for April 1.

That plot.ly library seems like a good topic.  I don't know much about it and I personally would like to know more.

There's no hard and fast rule for presentations, but generally you should go by what's interesting for you and at least potentially useful for others in the class.  For example, since we'll discuss Matplotlib through this lecture 

http://quant-econ.net/py/matplotlib.html

one idea would be to show how you would approach producing those graphs in plot.ly.  More generally, you might like to compare its functionality and approach to what we can find in Matplotlib. 

This is just a suggestion.

I also think its good if we take an academic and critical approach to the tools we discuss, and state not just good points but also the things that we perceive to be weaknesses.  For example, does it have decent LaTeX integration?  Might we regret investing time in learning it if we later discover that it isn't sufficiently customizable?  (For me it's also helpful to know if I should cross a given tool off my list of things to invest time in, even if it looks good on first impression.)

**Comment by jstac** [comment]
Regarding class presentations, I've made a schedule so please use it to help you choose a date and topic.

https://github.com/jstac/quantecon_nyu_2016/wiki/Presentation-Schedule

**Comment by FelipeAAlves** [comment]
HI John, 

Can I make the presentation on Heterogeneous Agents in cont time in April 29?

Best, Felipe

**Comment by jstac** [comment]
@FelipeAAlves Done.

**Comment by brandomagnani** [comment]
Hi John, I'd like to give a presentation on April 15 about one of the
scikit-learn libraries and maybe show some applications. (I was thinking
about cross validation or something in the regression package with an
application to stock prices, for example). Sounds good?

Best,
Brando

On Thu, Mar 3, 2016 at 5:01 PM, John Stachurski notifications@github.com
wrote:

> @FelipeAAlves https://github.com/FelipeAAlves Done.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jstac/quantecon_nyu_2016/issues/16#issuecomment-191984857
> .

## 

Best Regards,
Ildebrando Magnani

**Comment by jstac** [comment]
@brandomagnani Sounds good.  I've put you in the schedule.

**Comment by brandomagnani** [comment]
Thanks!
Il 03/mar/2016 19:57, "John Stachurski" notifications@github.com ha
scritto:

**Comment by jmorelli88** [comment]
Hi John, I was thinking on presenting on April 8th the Julia's version of the CompEcon package done by Spencer. I would probably combine it with some applications. Do you think that would be OK?

**Comment by jstac** [comment]
@jmorelli88 Yes, that sounds fine.  I've put you down. Note that Alberto is presenting on Interpolations.jl on the same day.

**Comment by jstac** [comment]
@lt1245 
@Abby27

I don't yet have you down for class presentations.  Apologies if you already suggested a date and I missed it.  Otherwise please choose one ASAP.  You can have March 25, April 1 or May 6 (but not both of you on the same date).

We still have no one presenting on [pandas](http://pandas.pydata.org/).  That would be a good choice.  You can get other ideas from reading the [course homepage](https://github.com/jstac/quantecon_nyu_2016).

You can see what others are presenting on here.

https://github.com/jstac/quantecon_nyu_2016/wiki/Presentation-Schedule

**Comment by lt1245** [comment]
Hi John,

we have talked about it and I will present pandas next class

**Comment by jstac** [comment]
Thanks everyone for letting me know your time and topic.  I'm looking forward to your talks.

Presentations should run for 20 minutes unless we agree otherwise, and I'll be relatively strict about time.  (This is how conferences run.  Please time yourself before hand so you know how long your talk is.)

Please focus on communication.  These are topics we all find interesting and useful.  You main job is to convey the information as clearly and effectively as you can.  

Please also take an academic approach to your topic.  Our job is not to promote, but rather to examine strengths and weaknesses.
